compiler/luci/pass/include/luci/Pass/ConvertNCHWToNHWCPass.h

```diff
@@ -35,11 +35,22 @@ namespace luci
  */
 struct ConvertNCHWToNHWCPass final : public logo::Pass
 {
+public:
+  ConvertNCHWToNHWCPass(bool preserve_input = false, bool preserve_output = false)
+    : _preserve_input(preserve_input), _preserve_output(preserve_output)
+  {
+    // Do nothing
+  }
+
   virtual ~ConvertNCHWToNHWCPass() = default;
 
   const char *name(void) const final { return "luci::ConvertNCHWToNHWCPass"; }
 
   bool run(loco::Graph *g) final;
+
+private:
+  bool _preserve_input;
+  bool _preserve_output;
```

Comment:
```suggestion
  bool _preserve_input = false;
  bool _preserve_output = false;
```

or make default constructor `delete` would be better

Reply:
Updated. Thanks.